variable/variable.cpp

```diff
@@ -182,6 +182,14 @@ void Variable::rename(const Dim from, const Dim to) {
     data().m_dimensions.relabel(dims().index(from), to);
 }
 
+/// Rename dims of a view. Does NOT rename dims of the underlying variable.
+void VariableConstView::rename(const Dim from, const Dim to) {
```

Comment:
It looks like this will be safe for later operations such as view slicing, but does not look to be tested. Have you verified?